easybuild/easyconfigs/f/FFmpeg/FFmpeg-3.1.3-intel-2016b.eb

```diff
@@ -14,7 +14,9 @@ source_urls = ['http://ffmpeg.org/releases/']
 dependencies = [
     ('NASM', '2.12.02'),
     ('zlib', '1.2.8'),
+    ('bzip2', '1.0.6'),
     ('x264', '20160614'),
+    ('libxcb', '1.11.1'),
```

Comment:
@FokkeDijkstra please use the `X11` bundle here, which includes `libxcb`: `(('X11', '20160819'),`